### PR TITLE
Add a revision number to the docName

### PR DESCRIPTION
**Pull request by fxlb**

### CODE REVIEW

File: draft-gharris-opsawg-pcap.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 <rfc category='info'
      ipr='trust200902'
-     docName="draft-gharris-opsawg-pcap">
+     docName="draft-gharris-opsawg-pcap-01">
```
Comment (guyharris) [inline draft-gharris-opsawg-pcap.xml]: -01, or -00?  [These guidelines](https://www.ietf.org/standards/ids/guidelines/#7) say "If the document is a new one (i.e., a version of "00") ...".

Comment (fxlb) [reply draft-gharris-opsawg-pcap.xml]: Yes ! Updated.

Comment (guyharris) [reply draft-gharris-opsawg-pcap.xml]: There's already a -01 that presumably @mcr submitted, so maybe we should go to -02 now.

**Comment by fxlb** [comment]
Link to the -01 ? pcap or pcapng ?

**Comment by guyharris** [comment]
Sorry, I'm mixing the two.  I'll fix up the pcapng document.

**Pull request merged**